app/controllers/assignments_controller.rb

```diff
@@ -321,6 +321,8 @@ def is_int(val)
   end
 
   def set_questions_graders
+    # FIXME: This is way too complicated.
+
```

Comment:
Probably should get moved out to something in lib which gets called from the model?